modules/gltf/editor/editor_scene_importer_gltf.cpp

```diff
@@ -69,4 +70,23 @@ Ref<Animation> EditorSceneFormatImporterGLTF::import_animation(const String &p_p
 	return Ref<Animation>();
 }
 
+void EditorSceneFormatImporterGLTF::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("set_gltf_extensions", "extensions"), &EditorSceneFormatImporterGLTF::set_gltf_extensions);
+	ClassDB::bind_method(D_METHOD("get_gltf_extensions"), &EditorSceneFormatImporterGLTF::get_gltf_extensions);
+	ADD_PROPERTY(PropertyInfo(Variant::ARRAY, "gltf_extensions", PROPERTY_HINT_ARRAY_TYPE,
+						 vformat("%s/%s:%s", Variant::OBJECT, PROPERTY_HINT_RESOURCE_TYPE, "GLTFDocumentExtension"),
+						 PROPERTY_USAGE_DEFAULT),
+			"set_gltf_extensions", "get_gltf_extensions");
+}
+
+void EditorSceneFormatImporterGLTF::set_gltf_extensions(TypedArray<GLTFDocumentExtension> p_extensions) {
+	ERR_FAIL_NULL(doc);
+	doc->set_extensions(p_extensions);
+}
+
+TypedArray<GLTFDocumentExtension> EditorSceneFormatImporterGLTF::get_gltf_extensions() {
+	ERR_FAIL_NULL_V(doc, Array());
+	return doc->get_extensions();
+}
```

Comment:
These may fail arbitrarily if doc is null.

I think this registration system should be rewrorked. A GLTFDocumentExtension should have a lifetime of just a single document, rather than permanent, so it can maintain state internally and avoid the complex options system below

Reply:
Removed the entire api on the EditorSceneFormatImporterGLTF class.